HW_3/task16.py

```diff
@@ -0,0 +1,18 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A.
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию).
+# Последняя строка содержит число X.
+
+import random
+
+n = int(input('Введите количество элементов массива '))
+lst = [random.randint(1, 9) for i in range(n)]
+print(lst)
+lst_1 = []
+x = int(input('Введите число '))
+count = 0
+for item in lst:
+    if item == x in lst:
```

Comment:
Зачем `in lst`? Нарушается логика работы
```suggestion
    if item == x:
```